auacm/angular/src/app/models/blog.ts

```diff
@@ -6,13 +6,15 @@ export class BlogPost {
   public body: string;
   public author: BlogAuthor;
   public resized: boolean;
+  public displayToggle: boolean;
```

Comment:
I feel like `displayToggle` isn't perhaps the best name... I just feel like saying `displayToggle` is `true` doesn't really tell you much information about how the blog post is being rendered. Perhaps `isExpanded` or something similar will be more descriptive?